### ISSUE TITLE
Linear Regression to Feedforward Neural Networks

### ISSUE THREAD

**Issue by vmohanan8** [vmohanan8/OpenCV#3]
- [ ] Linear Regression
- [ ] Logistic Regression 
- [ ] Gradient Descent
- [ ] Stochastic Gradient Descent
- [ ] Neural Networks - Backpropagation 
- [ ] Feedforward NN's
